idris-jvm-ffi/src/main/idris/IdrisJvm/JvmImport.idr

```diff
@@ -153,7 +153,7 @@ jvmImport cmd importList = do
     system cmd
     Right str <- readFile ".idrisjvmtypes"
         | Left err => pure (Error $ show err)
-    let errOrffiDescs = sequence $ parseJvmOutput <$> lines str
+    let errOrffiDescs = sequence $ parseJvmOutput <$> (filter (not . (== "")) $ lines str)
```

Comment:
We can do `filter (/= "")`

Reply:
O.k., I've simplified the filter expression - to make it consistent I choose to also change the two occurrences of this notation that have existed before. See new commit.

Reply:
Thank you.